cvat/apps/engine/models.py

```diff
@@ -292,6 +292,7 @@ class Meta:
 
 class Project(models.Model):
     name = SafeCharField(max_length=256)
+    description = SafeCharField(max_length=1400, default="")
```

Comment:
This is arbitrary length I propose, but maybe you think it would be good to change it.   
I am also not sure which field type you would recommend `SafeCharField` or regular `CharField`